setup.py

```diff
@@ -15,7 +15,7 @@
  render `Doxygen <http://www.doxygen.org>`__ xml output.
 '''
 
-requires = ['Sphinx>=2.0', 'docutils>=0.12', 'six>=1.9']
+requires = ['Sphinx>=3.0', 'docutils>=0.12', 'six>=1.9']
 
 if sys.version_info < (3, 5):
```

Comment:
Should this be `(3, 6)` now (and maybe error message indicates Sphinx 3.0 requires Python 3.6+)? Or am I misunderstanding the other changes?

Reply:
Hmm, it looks like 3.5 is only dropped in Sphinx 4 (https://www.sphinx-doc.org/en/master/changes.html). I have reenabled it in the workflows CI.